db/schema.rb

```diff
@@ -171,6 +171,10 @@
     t.uuid "school_id", null: false
     t.uuid "lead_provider_id", null: false
     t.uuid "cohort_id", null: false
+    t.string "status", default: "pending", null: false
+    t.string "reason_for_rejection"
```

Comment:
Think you need to regenerate the schema?

Reply:
👍 That's now been sorted